tests/test_serialization.py

```diff
@@ -183,6 +183,29 @@ def test_load_duplicate_keys_sub() -> None:
         os.unlink(fp.name)
 
 
+def test_load_merge_duplicates() -> None:
+    try:
+        with tempfile.NamedTemporaryFile(delete=False) as fp:
+            content = dedent(
+                """\
+                a: &A
+                    x: 1
+                b: &B
+                    y: 2
+                c:
+                    <<: *A
+                    <<: *B
+                    x: 3
+                    z: 1
+                """
+            )
+            fp.write(content.encode("utf-8"))
+        cfg = OmegaConf.load(fp.name)
```

Comment:
no need to go through a file. you can initialize OmegaConf directly from a yaml string:

```
cfg = OmegaConf.create(content)
```

At this point, also clean up test_load_duplicate_keys_top().
I think both tests belongs in test_create().